modules/op-release-notes-1-11.adoc

```diff
@@ -202,4 +202,36 @@ With this command, you can also specify a custom GitHub API URL by using the `--
+
+* Before this update, the `disable-ha` flag was not correctly passed to the `tekton-pipelines` controller, so the High Availability (HA) feature of {pipelines-title} could not be enabled. With this update, the `disable-ha` flag  is correctly passed and you can enable the HA feature as required.
+
+* Before this update, you could not set the URL for {tekton-hub} and {artifact-hub} for the hub resolver, so you could use only the preset addresses of {tekton-hub} and {artifact-hub}. With this update, you can configure the URL for {tekton-hub} and {artifact-hub} for the hub resolver, for example, to use a custom {tekton-hub} instance that you installed.
+
+* With this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
+
+* Before this update, the `tekton-pipelines-controller` component used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. With this update, the component uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
+
+* Before this update, when a user without write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as `skipped` in GitHub. With this update, {pac} CI/CD actions are shown as `Pending approval` in GitHub.
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. With this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
+
+* Before this update, metrics for the {pac} controller were not visible in the {product-title} developer console. With this update, metrics for the {pac} controller are displayed in the developer console.
+
+* Before this update, in {pipelines-title} 1.11.0, the Operator always installed {tekton-chains} and you could not disable installation of the {tekton-chains} component. With this update, you can set the value of the `disabled` parameter to `true` in the `TektonConfig` CR  to disable installation of {tekton-chains}.
+
+* Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` CR and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. With this update, if you upgrade from an older version of {pipelines-shortname}, {tekton-chains} configuration information is preserved.
```

Comment:
We need to add one note here 

NOTE:- Data will be preserved only if in previous versions Tekton Chains was installed in the targetNamespace i.e. where TektonConfig was installed (e.g. `openshift-pipelines`)